app/src/main/res/layout/item_media_preview.xml

```diff
@@ -110,6 +110,25 @@
             app:srcCompat="@drawable/ic_eye_24dp"
             app:tint="@color/white" />
 
+        <TextView
+            android:id="@+id/status_media_description_indicator"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textColor="?android:attr/textColorSecondary"
+            android:textSize="?attr/status_text_small"
+            android:background="@drawable/media_warning_bg"
+            android:contentDescription="@null"
+            android:paddingLeft="5dp"
+            android:paddingTop="3dp"
+            android:paddingRight="5dp"
+            android:paddingBottom="3dp"
+            android:layout_margin="3dp"
+            android:visibility="gone"
+            android:text="ALT"
+            app:layout_constraintRight_toRightOf="@+id/status_media_preview_container"
```

Comment:
Is RTL (reading direction) an issue here?

Reply:
Probably best to do `end` here

Reply:
Ok, also did it accordingly ("start") for status_sensitive_media_button.